include/osquery/sql.h

```diff
@@ -100,11 +116,26 @@ class SQL {
    */
   SQL(){};
 
- private:
+  // The key used to store hostname for annotateHostInfo
+  static const std::string kHostColumnName;
```

Comment:
We've been using `kSomethingSomething` for global const variables. Feel free to use the same for static members, but we could also `#define` this somewhere in a header?